### PR TITLE
Add school search page

### PR DESCRIPTION
**Pull request by ethanmills**
Ticket: https://trello.com/c/jgHC2LmF/46-prototype-school-search-page

### CODE REVIEW

File: app/assets/javascripts/school-search/data.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: false,
+    rural: false,
+    alreadyPartnered: false,
+    partnershipConfirmed: true
+  }
+]
+
+export const allMats = [
```
Comment (kicferk1) [inline app/assets/javascripts/school-search/data.js:82]: What does MAT stand for in this context?

File: app/assets/javascripts/school-search/data.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+export const allSchools = [
+  {
+    urn: 123456,
```
Comment (cfq) [inline app/assets/javascripts/school-search/data.js]: I'd prefer strings over integers for IDs.

File: app/assets/javascripts/school-search/data.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    name: 'Corfe Hills School',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
```
Comment (cfq) [inline app/assets/javascripts/school-search/data.js:5]: Could be useful to separate city and postcode for further filtering

File: app/views/school-search/index.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                label: "School",
+                id: "school-tab",
+                panel: { html: schoolTab }
+            },
+            {
+                label: "Multi-academy trust",
+                id: "mat-tab",
+                panel: {
+                html: matTab
+            }
```
Comment (kicferk1) [inline app/views/school-search/index.html]: Some indentation shenanigans here.

File: app/views/school-search/partials/filter-mats.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
+<h2 class="govuk-heading-m">Filter MATs</h2>
+{{ govukButton({
+    text: "Apply filters",
+    name: "apply-mat-filters"
+}) }}
+
+            {{ govukInput({
```
Comment (kicferk1) [inline app/views/school-search/partials/filter-mats.html]: ooof

Is there an auto-formatter for nunjucks out there?

File: app/assets/javascripts/school-search/data.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
```
Comment (cfq) [inline app/assets/javascripts/school-search/data.js:7]: excellent 👍

**Review by cfq — APPROVED** [review]

File: app/assets/javascripts/school-search/data.js (hunk 1 above)
Comment (ethanmills) [reply app/assets/javascripts/school-search/data.js:82]: multi academy trust

File: app/views/school-search/partials/filter-mats.html (hunk 5 above)
Comment (ethanmills) [reply app/views/school-search/partials/filter-mats.html]: Eesh, sorry :/
Not a good one that I could find. I'm using the Twig plugin for IntelliJ. VSCode does a slightly better job with it's Jinja plugin, but not much

File: app/assets/javascripts/school-search/data.js (hunk 3 above)
Comment (ethanmills) [reply app/assets/javascripts/school-search/data.js:5]: Location search was out of spec for the prototype, this is just for display. I can though, if you think it's worth it?

File: app/views/school-search/index.html (hunk 4 above)
Comment (ethanmills) [reply app/views/school-search/index.html]: The formatter helpfully corrects to this from anything more sensible. I'll one-line it

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**